sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/internal/utils/DatabaseUtils.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.extension.siddhi.execution.geovelocity.internal.utils;
+
+import com.zaxxer.hikari.HikariDataSource;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.framework.ServiceReference;
+import org.wso2.carbon.datasource.core.api.DataSourceService;
+import org.wso2.carbon.datasource.core.exception.DataSourceException;
+import org.wso2.extension.siddhi.execution.geovelocity.internal.exception.GeoVelocityException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+/**
+ * This class provides Database Util functionality.
+ */
+public class DatabaseUtils {
+
+    private static final DatabaseUtils instance = new DatabaseUtils();
+    private static final Log log = LogFactory.getLog(DatabaseUtils.class);
+
+    private DataSource dataSource = null;
+    private String dataSourceName;
+
+    private DatabaseUtils() {
+    }
+
+    public static DatabaseUtils getInstance() {
+        return instance;
```

Comment:
keep new line

Reply:
Fixed in https://github.com/wso2/samples-is/pull/265